install.html

```diff
@@ -202,6 +213,50 @@ <h4>7. Other INI files</h4>
 
       </ul>
 
+      <h2>2. Install Cypht within Tiki</h2>
```

Comment:
@rodriguezny shouldn't be the last way of installing cypht ?

Reply:
Updated!